src/cis2/mod.rs

```diff
@@ -338,7 +521,7 @@ impl Cis2Contract {
     /// * `queries` - A list queries to execute.
     pub async fn token_metadata(
         &mut self,
-        bi: &BlockIdentifier,
+        bi: impl IntoBlockIdentifier,
```

Comment:
No, becauase `&BlockIdentifier` implements `IntoBlockIdentifier`.

Reply:
I think I stated this also in the changelog.

Reply:
What is kind of breaking is if people used to do `&block.into()`. It would now fail because it would be confused about what type you want to convert to.